app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -86,18 +88,22 @@ open class BpkSnapshotTest {
     return AsyncSnapshot(testClass, testName)
   }
 
+  fun assumeVariant(vararg variants: BpkTestVariant) {
+    assumeThat(BpkTestVariant.current, Matchers.isOneOf(*variants))
+  }
+
   protected fun composed(
     size: IntSize = IntSize(width, height),
     background: Color = Color.Unspecified,
+    className: String = TestNameDetector.getTestClass().removePrefix("net.skyscanner.backpack."),
```

Comment:
rather than exposing these could we add another property, like `testParameters` and update the `getScreenshotName` method accordingly, so it also supports parameters? that way this solution also works for non-compose tests